### PR TITLE
Docsite, README: Add communication information

### PR DESCRIPTION
**Pull request by Andersson007**
Dear maintainers,
As a part of the [Consolidating Ansible discussion platforms initiative](https://forum.ansible.com/t/proposal-consolidating-ansible-discussion-platforms/6812), this PR adds the communication section template defined by the community to the README and the docsite. Similar PRs are now being raised across all the Ansible ecosystem projects.

Thank you

### CODE REVIEW

**Comment by Andersson007** [comment]
cc @oraNod

**Comment by Andersson007** [comment]
I have a couple of questions:
- Do you still use Libera.Chat?
- Any other groups/tags that should be mentioned?
- Should we maybe rename `Talk to us` to, say, more typical `Communication` or like on docs.ansible.com `Communicating with Ansible Dev Tools community`?

Any suggestions would be appreciated

Ready for review, thanks

File: docs/contributor-guide.md [hunk 1]
```diff
@@ -62,23 +62,29 @@ See the `tests/integration/test_container.py` for examples.
...
 
-For the full list of Ansible IRC and Mailing list, please see the [Ansible
-Communication] page. Release announcements will be made to the [Ansible
-Announce] list.
+  - [Get Help](https://forum.ansible.com/c/help/6): get help or help others. Please add appropriate tags if you start new discussions, for example the `devtools` tag.
+  - [Posts tagged with 'devtools'](https://forum.ansible.com/tag/devtools): subscribe to participate in project-related conversations.
+  - [Social Spaces](https://forum.ansible.com/c/chat/4): gather and interact with fellow enthusiasts.
+  - [News & Announcements](https://forum.ansible.com/c/news/5): track project-wide announcements including social events.
+
+- The Ansible [Bullhorn newsletter]: used to announce releases and important changes.
```
Comment (oraNod) [inline docs/contributor-guide.md:72]: @Andersson007 is there a missing link to the bullhorn?

Comment (oraNod) [reply docs/contributor-guide.md:72]: Oh nvm! Sorry I missed the reference at the end and forgot that you could do that sort of linking with MD. Was expecting a url in parentheses.

File: docs/contributor-guide.md [hunk 2]
```diff
@@ -62,23 +62,29 @@ See the `tests/integration/test_container.py` for examples.
...
 
 Please see the official [Ansible Community Code of Conduct].
 
-[discussions]: https://github.com/ansible/ansible-dev-tools/discussions
-[ansible communication]: https://docs.ansible.com/ansible/latest/community/communication.html
-[ansible announce]: https://groups.google.com/forum/#!forum/ansible-announce
+[Ansible communication guide]: https://docs.ansible.com/ansible/devel/community/communication.html
 [Ansible Community Code of Conduct]: https://docs.ansible.com/ansible/latest/community/code_of_conduct.html
 [creating your fork on github]: https://docs.github.com/en/get-started/quickstart/contributing-to-projects
+[Bullhorn newsletter]: https://docs.ansible.com/ansible/devel/community/communication.html#the-bullhorn
```
Comment (oraNod) [inline docs/contributor-guide.md:90]: @Andersson007 Maybe we should point to the forum for the bullhorn here too? https://forum.ansible.com/c/news/bullhorn/17

File: docs/contributor-guide.md (hunk 1 above)
Comment (Andersson007) [reply docs/contributor-guide.md:72]: I don't personally like this style, I'll change it, looks confusing to me too

File: docs/contributor-guide.md (hunk 2 above)
Comment (Andersson007) [reply docs/contributor-guide.md]: yep, as it now lives in the forum anyway, makes sense, done, thanks

File: docs/contributor-guide.md [hunk 3]
```diff
@@ -62,23 +62,28 @@ See the `tests/integration/test_container.py` for examples.
...
-For the full list of Ansible IRC and Mailing list, please see the [Ansible
-Communication] page. Release announcements will be made to the [Ansible
-Announce] list.
+  - [Get Help](https://forum.ansible.com/c/help/6): get help or help others. Please add appropriate tags if you start new discussions, for example the `devtools` tag.
+  - [Posts tagged with 'devtools'](https://forum.ansible.com/tag/devtools): subscribe to participate in project-related conversations.
+  - [Social Spaces](https://forum.ansible.com/c/chat/4): gather and interact with fellow enthusiasts.
+  - [News & Announcements](https://forum.ansible.com/c/news/5): track project-wide announcements including social events.
+  - [Bullhorn newsletter](https://docs.ansible.com/ansible/devel/community/communication.html#the-bullhorn): used to announce releases and important changes.
+
+- The Ansible [Bullhorn newsletter]: used to announce releases and important changes.
```
Comment (Andersson007) [inline docs/contributor-guide.md:73]:
```suggestion
```

**Comment by ssbarnea** [comment]
I am wondering if we could move these instructions inside the themes (mkdocs and sphinx), so we do not need to bother updating them each time we make a change)

**Comment by oraNod** [comment]
> I am wondering if we could move these instructions inside the themes (mkdocs and sphinx), so we do not need to bother updating them each time we make a change)

@ssbarnea Hey, I think a link to the forum would be useful in the theme navbar for sure. I think each project has slightly different versions of these changes though.

I submitted a corresponding PR to the core repo and bcoca had a similar comment: https://github.com/ansible/ansible/pull/83862#issuecomment-2310837325

Maybe we could just point everyone to the same communication page in the docs?

**Pull request merged**